lightning/src/ln/channelmanager.rs

```diff
@@ -731,18 +731,21 @@ pub type SimpleRefChannelManager<'a, 'b, 'c, 'd, 'e, M, T, F, L> = ChannelManage
 //  |                   |
 //  |                   |__`pending_background_events`
 //
-pub struct ChannelManager<M: Deref, T: Deref, K: Deref, F: Deref, L: Deref>
+pub struct ChannelManager<M: Deref, T: Deref, K: Deref, F: Deref, R: Deref, L: Deref>
 	where M::Target: chain::Watch<<K::Target as KeysInterface>::Signer>,
-        T::Target: BroadcasterInterface,
-        K::Target: KeysInterface,
-        F::Target: FeeEstimator,
-				L::Target: Logger,
+	      T::Target: BroadcasterInterface,
+	      K::Target: KeysInterface,
+	      F::Target: FeeEstimator,
+	      R::Target: Router,
+	      L::Target: Logger,
```

Comment:
Maybe a good opportunity to fix the indentation to only use tabs:

```rust
pub struct ChannelManager<M: Deref, T: Deref, K: Deref, F: Deref, R: Deref, L: Deref>
where
	M::Target: chain::Watch<<K::Target as KeysInterface>::Signer>,
	T::Target: BroadcasterInterface,
	K::Target: KeysInterface,
	F::Target: FeeEstimator,
	R::Target: Router,
	L::Target: Logger,
{
```

Likewise throughout.